java.base/src/main/java/java/lang/Thread$_patch.java

```diff
@@ -84,13 +84,13 @@ public class Thread$_patch {
     native void setPriority(final int priority);
     static native long nextThreadID();
 
-    @Add
+    @Add(unless = { Build.Target.IsWasm.class })
```

Comment:
I should be able to do it :) basically for now it would just mean:

````
boolean isPThread() { return isWasm(); }
```

but then it can be made as clever as needed

Reply:
needs https://github.com/qbicc/qbicc/pull/1436

Reply:
for some reason

    @Add(when = { Build.Target.IsPThreads.class })
    pthread_t thread;

is not equivalent to unless=wasm 🤨 

i.e. I am seeing a:

```
java.lang.NullPointerException: Cannot invoke "org.qbicc.type.definition.DefinedTypeDefinition.load()" because "jltDef" is null
```